components/form/FAutocompleteInput.vue

```diff
@@ -357,18 +353,10 @@ function isValidMatch() {
   return !inputValue.value || !!currentOptionMatched.value;
 }
 
-function handleSelectOption(optionValue: any) {
-  fieldValue.value = optionValue;
-  currentOptionMatched.value = props.options.find(option =>
-    equal(optionValue, option.value)
-  );
-
-  formatInputValue();
-
-  validate(fieldValue.value);
```

Comment:
I tried it, and no more validation is made when choosing an option, which isn't optimal experience in my opinion (can be tested with the `Error` template of storybook)

I think we need to fire a manual validation whenever `fieldValue` change (= an option was selected or input was cleared), do you agree ?

Reply:
Ok I missed that. I'll check